hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/util/Util.java

```diff
@@ -239,11 +239,11 @@ public void write(int b) {
     }
 
     public static String jobNameAndExecutionId(String jobName, long executionId) {
-        return "job '" + jobName + "', execution " + idToString(executionId);
+        return "job '" + jobName + "', execution '" + idToString(executionId) + "'";
```

Comment:
We put the name to quotes, but not ID. As in code, strings are quoted, not numbers.